distributions/openhab/src/main/resources/bin/update.ps1

```diff
@@ -1,4 +1,27 @@
-#Requires -Version 5.0
+#Requires -Version 4.0
+function DownloadFiles {
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$DownloadSource,
+        [Parameter(Mandatory=$true)]
+        [string]$Outputfile
+    )
+    $client = New-Object System.Net.WebClient
+    try {
+        $client.DownloadFileAsync($DownloadLocation,$Outputfile)
+        while ($client.IsBusy)
+        {
+            Write-Host "Downloading file..."
```

Comment:
`Invoke-WebRequest` has poor performance - I suppose it's not recommended for downloading large binaries.

Using `WebClient.DownloadFileAsync` is better, but you should really replace the while loop by the event handling/`Write-Progress` in the StackOverflow link above, and it would report progress in a standard way, much like `Start-BitsTransfer` (I hope).